libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -256,6 +256,10 @@ void GCS_MAVLINK::send_battery_status(const uint8_t instance) const
     } else {
         consumed_wh = -1;
     }
+
+    uint8_t charge_state;
+    battery.get_mavlink_charge_state(instance, charge_state);
```

Comment:
check return, and use UNDEFINED if unavailable

Reply:
Fixed so that it returns MAV_BATTERY_CHARGE_STATE instead and CHARGE_STATE_UNDEFINED when unavailable